net-wireless/python-wifi/python-wifi-0.3.1-r2.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2012 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: /var/cvsroot/gentoo-x86/net-wireless/python-wifi/python-wifi-0.3.1-r1.ebuild,v 1.3 2012/02/25 11:53:52 patrick Exp $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+DISTUTILS_SINGLE_IMPL=1
```

Comment:
Hmm, i think i've wrongly interpret SUPPORT_PYTHON_ABIS="1" from the old ebuild. I'll gonna remove DISTUTILS_SINGLE_IMPL.